# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/painting/nsDisplayList.h
# Commit: a73e041f1f1d
# Full Hash: a73e041f1f1dc2537292b5e3111efda0899b53af
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-05-18 09:45:31
# Regressor Bug: 1540737
# File Overlap Count: 1
# Description:
#   Bug 1540737 - Don't allow constructing an AutoBuildingDisplayList with a new frame without requiring explicit rects in the new coordinate space. r=miko
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D114675
# ==============================================================================

diff -r 81f1a9d1a5e1 -r a73e041f1f1d layout/painting/nsDisplayList.h
--- a/layout/painting/nsDisplayList.h	Mon May 17 23:53:44 2021 +0000
+++ b/layout/painting/nsDisplayList.h	Mon May 17 23:53:44 2021 +0000
@@ -1104,11 +1104,6 @@
    */
   class AutoBuildingDisplayList {
    public:
-    AutoBuildingDisplayList(nsDisplayListBuilder* aBuilder, nsIFrame* aForChild)
-        : AutoBuildingDisplayList(
-              aBuilder, aForChild, aBuilder->GetVisibleRect(),
-              aBuilder->GetDirtyRect(), aForChild->IsTransformed()) {}
-
     AutoBuildingDisplayList(nsDisplayListBuilder* aBuilder, nsIFrame* aForChild,
                             const nsRect& aVisibleRect,
                             const nsRect& aDirtyRect)
